RoaringBitmap/src/main/java/org/roaringbitmap/Container.java

```diff
@@ -263,6 +263,13 @@ public boolean contains(Container subset) {
    */
   public abstract void deserialize(DataInput in) throws IOException;
 
+  /**
+   * Deserialize the container
+   * @param byteBuffer the ByteBuffer
+   * @throws IOException Signals that an I/O exception has occurred.
+   */
+  public abstract void deserialize(ByteBuffer byteBuffer) throws IOException;
```

Comment:
The reason this method didn't exist before was that the cardinalities and bitmap types are available in the serialised format of `RoaringArray` so they can be constructed there.